tests/test_api_security.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+"""
+Tests for the CAG API ViewSets.
+"""
+
+from __future__ import absolute_import, unicode_literals
+
+import pytest
+from django.views import View
+from course_access_groups import views as cag_views
+from course_access_groups.security import CommonAuthMixin
+
+
+class TestCourseAccessGroupsViewSet(object):
+    """
+    Tests for the CourseAccessGroupsViewSet APIs.
+    """
+
+    view_classes = [
+        variable for variable in [
```

Comment:
Maybe use `klass` instead of `variable` because it is more descriptive and a common pattern?

Reply:
Thanks for the question John. I think it's worth moving to a helper to make it more obvious, so I did.